ansible/roles/captain/vars/main.yml

```diff
@@ -1,5 +1,9 @@
 deploy_env: captain
 # supervisor_task_conf: "{{ service_home }}/supervisor"
+#
+# Defined in config directory:
+# softlayer_vpn_username: ...
+# softlayer_vpn_password: ...
```

Comment:
@benrudolph: I'm assuming you copied these credentials from keepass into the config repo. I think this was one of the reasons we did not implement deploy from remote host in the past. I had tried to get it setup, but we decided against it at the time because it required putting login credentials on a file system on a remote server somewhere, which we were uncomfortable with at the time. Did you discuss this with other members of the team, or are you going with it's easier to ask forgiveness than permission?